package.json

```
@@ -53,26 +54,30 @@
         "react-redux": "^8.0.0",
         "react-resizable": "^3.0.4",
         "react-router-dom": "^6.0.0",
-        "react-scripts": "^5.0.0",
-        "react-virtualized": "^9.21.2",
+        "react-virtualized": "^9.22.5",
         "react-window": "^1.8.5",
         "reconnecting-websocket": "^4.4.0",
         "redux": "^4.0.5",
         "typeface-roboto": "^1.0.0",
         "typescript": "^5.0.4",
+        "uuid": "^3.4.0",
         "yup": "^1.0.0"
     },
     "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test --watchAll=false --transformIgnorePatterns \"node_modules/(?!@gridsuite/commons-ui)/\"",
-        "eject": "react-scripts eject"
+        "start": "vite",
+        "build": "tsc && vite build",
+        "serve": "vite preview",
+        "test": "jest",
+        "lint": "eslint . --ext js,mjs,jsx,ts,mts,tsx --max-warnings 0"
```

`./src` to not scan `node_modules`?
```suggestion
        "lint": "eslint ./src --ext js,mjs,jsx,ts,mts,tsx --max-warnings 0"
```

---

Nope and some other dot starting files